src/backend/core/management/commands/update_suspicious_item_file_hash.py

```diff
@@ -0,0 +1,27 @@
+"""Update suspicious item file hash"""
+
+from django.core.management.base import BaseCommand
+
+from core.models import Item, ItemUploadStateChoices
+from core.tasks.item import update_suspicious_item_file_hash
+
+
+class Command(BaseCommand):
```

Comment:
I don't see any cron job to start this command, what do you think regarding its execution ? One time exec by hand, cron, etc ?